index.html

```diff
@@ -19,220 +20,257 @@
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <div class="job-desc">
+        <h2>I'm a Software Developer</h2>
+      </div>
+      <div class="intro">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+      </div>
+    </div>
+      <div class="icons-container">
+        <ul>
+          <li><a><img src="images/github.png" alt="github"/></a></li>
+          <li><a><img src="images/linkedin.png" alt="linkedin"/></a></li>
+          <li><a><img src="images/angel list.png" alt="angel-list"/></a></li>
+          <li><a><img src="images/twitter.png" alt="twitter"/></a></li>
+          <li><a><img src="images/medium.png" alt="medium"/></a></li>
+        </ul>
+        </div>
+    </section>  
+      <!-- Recent-Works -->
+    <div class="recent-works">
 
-             <ul class="techstack-container">
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
-                    
-            </ul>
-            <button class="project-button">See Project</button>     
-
-            </div>
-         <div class="work">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            <ul class="techstack-container">
-              <li>html</li>
-              <li>Javascript</li>
-              <li>Css</li>
-            </ul>
-            <button class="project-button">See Project</button>
-          </div>
+        <h2>My Recent Work</h2>
+      <hr>  
+    </div>
     
-          <div class="work">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            <ul class="techstack-container">
-              <li>html</li>
-              <li>Javascript</li>
-              <li>Css</li>
-            </ul>
-            <button class="project-button">See Project</button>
-          </div>
-    
-          <div class="work">
-            <h2>Profesional Art Printing Data</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            <ul class="techstack-container">
+    <section class="works">
```

Comment:
- [ ] As requested by the previous reviewer, kindly add a bit of space at the bottom of the list of technologies for each project card.
- [Optional] Consider removing the white border added to the technologies and add a background color as in the Figma template. You are expected to stick to the design template as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/222882485-5adfc092-1ab4-4c0f-956c-9e435caf1a17.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/222882534-49361a93-c574-4918-8777-e41e016ea355.png)


- [ ] Also on the desktop view, kindly remove the white space between the project image and the button as requested by the reviewer.
- [Optional] Consider adding the border around the project card as in the figma template
- [Optional] Kindly consider making the project image and the project button the same width, please.
### Figma
![image](https://user-images.githubusercontent.com/94551484/222883312-051c0710-9db9-4238-a298-181c5b816753.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/222883441-6251165b-af18-4c47-a46a-4f796f480775.png)